aws-android-sdk-iot/src/test/java/android/util/Log.java

```diff
@@ -0,0 +1,30 @@
+package android.util;
+
+/**
+ * Avoid the need to use PowerMockito for Log!
```

Comment:
Robolectric should be able to do Android `Log`; I think we can just get rid of PowerMockito.

Reply:
Unfortunately i was getting `java.lang.RuntimeException: Method isLoggable in android.util.Log not mocked. See http://g.co/androidstudio/not-mocked for details.`.

I intentionally wrote this mock `Log` class to avoid using PowerMockito so IoT doesn't have any new dependency :)